### DISCUSSION TITLE
dv.pages() for pages linked to the current page

### DISCUSSION THREAD

**Discussion by thecookiemomma** [mdelobelle/metadatamenu#564] [Q&A] [answered]
I'm very new to dvjs .  I would like to set up a lookup field that only looks at pages that link to the current page. I have a field called "sources" that I put links to the pages / info that I use to develop that note.  I'd like the source pages to pull up where they're referenced.  I'm sure it's absolutely something simple, but with my limited skills, I have no clue.  Thanks.

**Comment by mdelobelle** [comment, marked as answer]
Hi if I understand correctly what you want to do:

Create a `Referencing` Lookup field with `Sources` as the target field and `dv.pages()` as the query
It will look for all pages in your vault for Sources fields and put into referencing only the links that point to the note where the `Referencing` field is
<img width="473" alt="image" src="https://github.com/mdelobelle/metadatamenu/assets/1233240/2448bf21-429f-4983-bb13-ac92a7c6f6f9">

<img width="935" alt="image" src="https://github.com/mdelobelle/metadatamenu/assets/1233240/0397a6a6-571a-4a0c-9225-8eba8f7a028e">

**Comment by thecookiemomma** [comment]
Derp that worked. Thank you so much. You are amazing.
